python/pyspark/context.py

```diff
@@ -456,7 +463,7 @@ def _dictToJavaMap(self, d):
         jm = self._jvm.java.util.HashMap()
         if not d:
             d = {}
-        for k, v in d.iteritems():
+        for k, v in d.items():
```

Comment:
The overhead of items() for dict with 1 million items is just 80 ms, so I'd like go with it.

```
davies@localhost:~/work/spark$ python -m timeit -s "d=dict(zip(range(1<<20), range(1<<20)))" -c "sum(1 for _ in d.iteritems())"
10 loops, best of 3: 80.7 msec per loop
davies@localhost:~/work/spark$ python -m timeit -s "d=dict(zip(range(1<<20), range(1<<20)))" -c "sum(1 for _ in d.items())"
10 loops, best of 3: 162 msec per loop
```